frontend/Dockerfile

```diff
@@ -6,29 +6,27 @@ COPY . .
 # Install dependencies
 RUN yarn install --frozen-lockfile
 
-# Copy environment variables file if it doesn't exist
-RUN test -e .env || cp .env.example .env
-
-ARG VITE_USE_RELATIVE_BASE_URL
+# Define build arguments
+ARG VITE_DOCS_QA_DELETE_COLLECTIONS
+ARG VITE_DOCS_QA_ENABLE_REDIRECT
+ARG VITE_DOCS_QA_MAX_UPLOAD_SIZE_MB
 ARG VITE_DOCS_QA_STANDALONE_PATH
+ARG VITE_GTAG_ID
 ARG VITE_QA_FOUNDRY_URL
 ARG VITE_USE_LOCAL
-ARG VITE_GTAG_ID
+ARG VITE_USE_RELATIVE_BASE_URL
 ARG VITE_CARBON_API_KEY
-ENV VITE_USE_RELATIVE_BASE_URL=${VITE_USE_RELATIVE_BASE_URL} \
-    VITE_DOCS_QA_STANDALONE_PATH=${VITE_DOCS_QA_STANDALONE_PATH}
-RUN if [ -n "$VITE_QA_FOUNDRY_URL" ]; then \
-    export VITE_QA_FOUNDRY_URL=${VITE_QA_FOUNDRY_URL}; \
-    fi
-RUN if [ -n "$VITE_USE_LOCAL" ]; then \
-    export VITE_USE_LOCAL=${VITE_USE_LOCAL}; \
-    fi
-RUN if [ -n "$VITE_GTAG_ID" ]; then \
-    export VITE_GTAG_ID=${VITE_GTAG_ID}; \
-    fi
-RUN if [ -n "$VITE_CARBON_API_KEY" ]; then \
-    export VITE_CARBON_API_KEY=${VITE_CARBON_API_KEY}; \
-    fi
```

Comment:
Why remove the conditionals here? What if these are provided in build args, do we still get the correct behavior?

Reply:
yes they work